service/src/integrationTest/kotlin/fi/espoo/evaka/vasu/VasuTemplateTest.kt

```diff
@@ -0,0 +1,134 @@
+// SPDX-FileCopyrightText: 2017-2021 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+package fi.espoo.evaka.vasu
+
+import fi.espoo.evaka.shared.domain.BadRequest
+import fi.espoo.evaka.shared.domain.FiniteDateRange
+import fi.espoo.evaka.shared.domain.HelsinkiDateTime
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+import org.junit.jupiter.api.assertThrows
```

Comment:
These will fail after rebase. Maybe we should also put my integration test PR to master first and then merge these tests in.